docs/home/whats-new-2-18.md

```diff
@@ -0,0 +1,116 @@
+    * Shared between multiple scopes&mdash;unlike other Run:ai data sources, data volumes can be shared across projects, departments, or clusters. This promotes data reuse and collaboration within your organization.
+    * Coupled to workloads in the submission process&mdash;similar to other Run:ai data sources, Data volumes can be easily attached to AI workloads during submission, specifying the data path within the workload environment.
+  
+    For more information, see [Data Volumes](../developer/admin-rest-api/data-volumes.md).
+
+* <!-- TODO better explanation RUN-16917/RUN-19363 Expose secrets in workload submission -->Added new data source type. Run:ai now allows you to configure a *Credential* (Secret) as a data source.
+
+#### Credentials
+
+* <!-- TODO better explanation RUN-16917/RUN-19363 Expose secrets in workload submission -->Added new *Generic secret* to the *Credentials*.
+
+#### SSO
+
+* <!-- TODO Change ticket numbers and description RUN-16859/RUN-16860-->Run:ai now supports SSO using OpenShift v4 (which is based on OIDC). Before using OpenShift, you must first define OAuthClient which interacts with OpenShift's OAuth server to authenticate users and request access tokens. For more information, see [Single Sign-On](../admin/runai-setup/authentication/sso/).
+
+* <!-- RUN-16788/RUN-16866 - OIDC Scopes -->OIDC scopes have been added to the authentication request. Scopes are used to specify what access privileges are being requested for access tokens. The scopes associated with the access tokens determine what resource are available when they are used to access OAuth 2.0 protected endpoints. Protected endpoints may perform different actions and return different information based on the scope values and other parameters used when requesting the presented access token. For more information, see [UI configuration](../admin/runai-setup/authentication/sso/#step-1-ui-configuration).
+
+#### Ownership protection
+
+* <!-- TODO Need to confirm text RUN-19098/RUN-19557 Need to add link -->Ownership protection in Run:ai ensures that only authorized users can delete or modify workloads. This feature is designed to safeguard important jobs and configurations from accidental or unauthorized modifications by users who did not originally create the workload.
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform.
+
+### Feature deprecations
+
+Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative.
+
+* Command Line Interface (CLI)&mdash;from cluster version 2.18 and higher, the *Legacy CLI* is deprecated. The *Legacy CLI* is still available for use on clusters that are 2.18 or higher, but it is recommended that you use the new *Improved CLI*.
```

Comment:
CLI deprecation. We do not have a GA version. We need to fully support if for at least 2 versions... This is not in alignment with our policy. Plus CLI deprecation is a huge deprecation. It is not a "feature".
@lavianalon , where are we with aligning internally and externally on the deprecation?

Reply:
WDYM "we don't have a GA version?".  what's the problem of declaring the CLI as deprecated? we are keeping it as long as we like but why not declare it as deprecated?